tools/run_danesfield.py

```diff
@@ -230,17 +230,18 @@ def main(args):
     #############################################
 
     input_paths = []
-    for root, dirs, files in itertools.chain(os.walk(config['paths']['imagery_dir']),
-                                             os.walk(config['paths']['rpc_dir'])):
+    for root, dirs, files in itertools.chain(os.walk(config['paths']['imagery_dir'])
+                                             #os.walk(config['paths']['rpc_dir'])
+                                             ):
         input_paths.extend([os.path.join(root, f) for f in files])
 
     collection_id_to_files = collate_input_paths(input_paths)
 
     # Prune the collection
     incomplete_ids = []
     for prefix, files in collection_id_to_files.items():
-        if 'msi' in files and ensure_complete_modality(files['msi'], require_rpc=True) and \
-           'pan' in files and ensure_complete_modality(files['pan'], require_rpc=True):
+        if 'msi' in files and ensure_complete_modality(files['msi'], require_rpc=False) and \
+           'pan' in files and ensure_complete_modality(files['pan'], require_rpc=False):
```

Comment:
We could use that same rpc flag mentioned above in this call

Reply:
The flag was added. It's based on the presence of rpc_dir in input.ini